pygpcca/_gpcca.py

```diff
@@ -1039,7 +1056,7 @@ def input_distribution(self) -> np.ndarray:
         return self._eta
 
     @property
-    def n_metastable(self) -> Optional[int]:
+    def n_macrostates(self) -> Optional[int]:
```

Comment:
Is this naming convention used consistently throughout pyGPCCA?

Reply:
Not 100% sure. At least in `_pygpcca` I checked it.

Reply:
`n_metastable`/`n_macrostates` is indeed only used in _pygpcca.